dist/browser.js

```diff
@@ -1,194 +1,213 @@
-/*
-The SDK interface for games to interact with Rune.
-*/
-var Rune = {
-    // External properties and functions
-    version: "1.4.5",
-    init: function (input) {
-        // Check that this function has not already been called
-        if (Rune._doneInit) {
-            throw new Error("Rune.init() should only be called once");
-        }
-        Rune._doneInit = true;
-        // Check that game provided correct input to SDK
-        var _a = input || {}, startGame = _a.startGame, resumeGame = _a.resumeGame, pauseGame = _a.pauseGame, getScore = _a.getScore;
-        if (typeof startGame !== "function") {
-            throw new Error("Invalid startGame function provided to Rune.init()");
-        }
-        if (typeof resumeGame !== "function") {
-            throw new Error("Invalid resumeGame function provided to Rune.init()");
-        }
-        if (typeof pauseGame !== "function") {
-            throw new Error("Invalid pauseGame function provided to Rune.init()");
-        }
-        if (typeof getScore !== "function") {
-            throw new Error("Invalid getScore function provided to Rune.init()");
-        }
-        Rune._validateScore(getScore());
-        // Initialize the SDK with the game's functions
-        Rune._startGame = startGame;
-        Rune._resumeGame = resumeGame;
-        Rune._pauseGame = pauseGame;
-        Rune._getScore = getScore;
-        // When running inside Rune, runePostMessage will always be defined.
-        if (window.postRuneEvent) {
-            window.postRuneEvent({ type: "INIT", version: Rune.version });
-        }
-        else {
-            Rune._mockEvents();
-        }
-    },
-    gameOver: function () {
+var Rune = (function () {
```

Comment:
`var Rune` seems to be defined both here and below? Might be good to only define it once? :)

Reply:
Discussed in a voice chat. Since everything is inside a function, the naming of variables inside does not matter.